### PR TITLE
perf(utils): utilize axel for faster, multi-threaded downloads (#92)

### PR DESCRIPTION
**Pull request by imtipi**

### CODE REVIEW

**Comment by imtipi** [comment]
currently, when I download snapshot from Cloudflare, I have an unstable connection and can't download it at full speed, and I use axel can mitigate the issue.
download progress command changed to du -b because axel will generate a .st file for continue download the file

**Comment by mkostrikin** [comment]
axel looks good, but very verbosibe.
```
[ 24%]  .......... .......... .......... .......... ..........  [48311.5KB/s]
[ 24%]  .......... .......... .......... .......... ..........  [48312.6KB/s]
[ 24%]  .......... .......... .......... .......... ..........  [48313.8KB/s]
[ 24%]  .......... .......... .......... .......... ..........  [48313.2KB/s]
```
While only this should report about progress.
```
while true ; do sleep 60; echo...
```

File: scripts/utils.sh [hunk 1]
```diff
@@ -15,9 +15,9 @@ function extract() {
 #   out: Location to download the file to.
 function download() {
   SIZE=$(curl -sI $1 | grep -i Content-Length | awk '{print $2}')
-  (while true ; do sleep 60; echo "$(ls -l $2 | awk -v size=$SIZE '{printf "Download Progress: %.2f%%\n", $5/size*100}')"; done) &
+  (while true ; do sleep 60; echo "$(du -b $2 | awk -v size=$SIZE '{printf "Download Progress: %.2f%%\n", $1/size*100}')"; done) &
```
Comment (mkostrikin) [inline scripts/utils.sh:18]:
```suggestion
  (while true ; do sleep 5; echo "$(du -b $2 | awk -v size=$SIZE '{printf "Download Progress: %.2f%%\n", $1/size*100}')"; done) &
```
60 sec is too much.

**Comment by smartcontracts** [comment]
Merged another fix that uses `aria2c`. Can you see if that works well? If it's good enough then I'll use that, otherwise will explore using a multi-threaded downloader.

**Comment by imtipi** [comment]
close due to already switch to snap sync and arid2c

**Pull request closed without merging**